types/wlr_linux_dmabuf.c

```diff
@@ -207,15 +207,16 @@ static void params_create_common(struct wl_client *client,
 			// Skip checks if kernel does no support seek on buffer
 			continue;
 		}
-		if (buffer->attributes.offset[i] >= size) {
+		if (buffer->attributes.offset[i] > size) {
 			wl_resource_post_error(params_resource,
 				ZWP_LINUX_BUFFER_PARAMS_V1_ERROR_OUT_OF_BOUNDS,
 				"invalid offset %i for plane %d",
 				buffer->attributes.offset[i], i);
 			goto err_out;
 		}
 
-		if (buffer->attributes.offset[i] + buffer->attributes.stride[i]	> size) {
+		if (buffer->attributes.offset[i] + buffer->attributes.stride[i] > size ||
+				buffer->attributes.stride[i] == 0) {
```

Comment:
That makes sense, also I wonder if we should reject stride < width?

Reply:
I'd rather not do that because there are obscure formats which have less than one byte per pixel.

Reply:
Yeah, it would only work for plane 0